gammapy/modeling/tests/test_fit.py

```diff
@@ -180,6 +180,25 @@ def test_optimize(backend):
     assert len(result.trace) == result.nfev
 
 
+@pytest.mark.parametrize("method", ["MINUIT", "SCIPY"])
```

Comment:
Should we not add a test about the similarity of the fitted resultats between different backends, ie scipy, iminuit, 3ml? This high level test is quite important